src/components/molecules/NavSearchBar/NavSearchResult.tsx

```diff
@@ -25,17 +31,13 @@ export const NavSearchResult: React.FC<NavSearchResultProps> = ({
     [image]
   );
 
-  const imageClasses = useMemo(
-    () =>
-      classNames("NavSearchResult__image", {
-        NavSearchResult__avatar: isAvatar,
-      }),
-    [isAvatar]
-  );
-
   return (
     <div className="NavSearchResult font-size--small" onClick={onClick}>
-      <div className={imageClasses} style={imageStyles} />
+      {isTruthy(user) ? (
```

Comment:
We probably could have just done `user ? (` (eg. without the `isTruthy`) here

Reply:
fixed: https://github.com/sparkletown/sparkle/pull/1350/commits/ce81df56da1ee5e60ad3e28b8d7ad395f7556447